docs/source/usage.rst

```diff
@@ -28,7 +28,7 @@ Look-up via registry:
 
     import fsspec
 
-    fs = fsspec.filesystem('file')
+    fs = fsspec.filesystem('local')
```

Comment:
I am OK to allow the "local" alias, but I don't want to change the default recommendation here